client/packages/plugins/backendCommon/utils.ts

```diff
@@ -45,3 +45,8 @@ export const fromSqlDateTime = (datetime: string) => {
   const withoutMillisecods = datetime.split('.')[0] || datetime;
   return new Date(`${withoutMillisecods.split(' ').join('T')}Z`);
 };
+
+export const toNaiveDateTime = (date: Date) => {
```

Comment:
This one is needed for creating message, we are using repository so need to provide correct format for sync message row